docs/Branching.md

```diff
@@ -134,12 +134,6 @@ Where a developer's work will be performed when implementing changes for a <span
       <span style="font-weight:bold">Branching Diagram</span>
    </div>
 
-  ![BranchingDiagram](./BranchingDiagram-DarkMode-v1.1.png#gh-dark-mode-only)
-  ![BranchingDiagram](./BranchingDiagram-LightMode-v1.1.png#gh-light-mode-only)
+  ![BranchingDiagram](https://github.com/KinsonDigital/.github/blob/master/docs/BranchingDiagram-DarkMode-v1.1.png)
```

Comment:
Refactor to add github preprocessor to the end of the URL

```suggestion
  ![BranchingDiagram](https://github.com/KinsonDigital/.github/blob/master/docs/BranchingDiagram-DarkMode-v1.1.png#gh-dark-mode-only)
```

Reply:
**Extra Info:**
This **#gh-dark-mode-only** section of text at the end of the URL is special syntax that only GitHub understands.  What it does is show the image in the markdown but only if the user has his GitHub account settings set to dark mode for GitHub.